### PR TITLE
changes date format: from bigint to timestamp

### PR DESCRIPTION
**Pull request by mk4111**
Linked to [this issue in ce100](https://github.com/emfoundation/ce100-app/issues/433)

Date can be automatically inputted by postgres, so have changed our queries and table set up to do so.

### CODE REVIEW

**Comment by mk4111** [comment]
@JMurphyWeb and @SimonLab - once you have time could you please have a look at these? Thank you!

File: tests/setup/challenges-data.test.js [hunk 1]
```diff
@@ -15,8 +14,8 @@ var challenge = [{
 var query = challengesData(challenge);
 
 var expected = 'INSERT INTO challenges '
```
Comment (SimonLab) [inline tests/setup/challenges-data.test.js]: It would be good to add a test to check the value of the date when a challenge is added

Comment (mk4111) [inline tests/setup/challenges-data.test.js]: Thank you so much for checking!!!

I'll write a test for that now!

Comment (SimonLab) [inline tests/setup/challenges-data.test.js]: You can call your endpoint which add a challenge - determine which id the challenge has been inserted to, then call  the getById endpoint or function to check the value of the data. I post this comment on the wrong part of the code sorry, this test might be better added to the one which add a challenge with the server.inject on "/"

**Comment by JMurphyWeb** [comment]
@SimonLab @mk4111 if this is a change to the mock data or data structure, then it should be updated in the other plugins as well. I don't know if it does affect other plugins though, but it looks like the mock data should be changed in each plugin and the main app.

**Comment by SimonLab** [comment]
@JMurphyWeb yes agree, @mk4111 do you mind to update and test the other plugins and application with the new mock data. Let us know if any blockers or issues are appearing, I don't think this feature is high priority at the moment and I'm still not sure about all the side effects it can have on the other plugins and data.

**Pull request merged**